pkgs/by-name/pr/primecount/package.nix

```diff
@@ -1,21 +1,23 @@
 { lib
-, stdenv
-, fetchFromGitHub
 , cmake
+, fetchFromGitHub
 , primesieve
+, stdenv
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "primecount";
   version = "7.10";
 
   src = fetchFromGitHub {
     owner = "kimwalisch";
     repo = "primecount";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-z7sHGR6zZSTV1PbL0WPGHf52CYQ572KC1yznCuIEJbQ=";
   };
 
+  outputs = [ "out" "dev" "lib" "man" ];
```

Comment:
I guess this breaks openturns: https://hydra.nixos.org/job/nixpkgs/trunk/openturns.x86_64-linux

@AndersonTorres Could you have a look?